src/components/AppNavigation/AppNavigation.vue

```diff
@@ -109,9 +109,14 @@ export default {
 		 */
 		toggleNavigation(state) {
 			this.open = state || !this.open
-			emit('navigation-toggled', {
-				open: this.open,
-			})
+			const bodyStyles = getComputedStyle(document.body)
+			const animationLength = parseInt(bodyStyles.getPropertyValue('--animation-quick')) | 100
```

Comment:
Damn, it should be `||`

Reply:
Indeed. Weird that this doesn't give a build error.

Reply:
Because it's valid, it's a binary thing